TestAssets/TestProjects/TestLibraryWithProjectFileWithoutFrameworks/project.json

```diff
@@ -0,0 +1,8 @@
+{
+    "dependencies": {
+        "Microsoft.VisualStudio.Web.CodeGeneration.Tools" : {
+            "version": "1.0.0-preview2-final",
+            "type": "build"
+        }
+    }
+}
```

Comment:
This likely needs to go in nonrestoredprojects. Does restore work without frameworks?